Octokit/Http/RateLimit.cs

```diff
@@ -2,17 +2,22 @@
 using System.Collections.Generic;
 using System.Runtime.Serialization;
 using Octokit.Helpers;
+using System.Diagnostics;
+using System.Globalization;
+using Octokit.Internal;
 
 namespace Octokit
 {
 #if !NETFX_CORE
     [Serializable]
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
```

Comment:
Will do.  I changed all this when I thought it was causing the appveyor problems (there was reasoning behind my thinking but I won't bore you)